### PR TITLE
Generate docs for /acs

### PR DESCRIPTION
**Pull request by razor-x**

### CODE REVIEW

File: docs/api/acs/access_groups/add_user.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+Type: `string`\
+Required: Yes
+
+ID of the desired access group.
+
+***
+
+### `acs_user_id`
+
+Type: `string`\
```
Comment (DebbieAtSeam) [inline docs/api/acs/access_groups/add_user.md:132]: @razor-x Is this backslash OK?

File: docs/api/acs/systems/README.md [hunk 2]
```diff
@@ -103,7 +103,9 @@ Possible enum values:
 - `assa_abloy_credential_service`
 - `latch_building`
 - `dormakaba_community`
+- `legic_connect`
```
Comment (DebbieAtSeam) [inline docs/api/acs/systems/README.md:106]: @razor-x We're OK with these values showing before the corresponding integration has been released, right?

File: docs/SUMMARY.md [hunk 3]
```diff
@@ -162,40 +162,40 @@
...
-    * [Create a Credential for a User](api-clients/acs/credentials/create.md)
-    * [Assign a Credential to a User](api-clients/acs/credentials/assign.md)
-    * [List Credentials](api-clients/acs/credentials/list.md)
-    * [Get a Credential](api-clients/acs/credentials/get.md)
-    * [Update a Credential](api-clients/acs/credentials/update.md)
-    * [Unassign a Credential from a User](api-clients/acs/credentials/unassign.md)
-    * [Delete a Credential](api-clients/acs/credentials/delete.md)
-    * [List Accessible Entrances](api-clients/acs/credentials/list\_accessible\_entrances.md)
+  * [Users](api/acs/users/README.md)
+    * [Create a User](api/acs/users/create.md)
```
Comment (razor-x) [inline docs/SUMMARY.md:171]:
```suggestion
    * [Create an ACS User](api/acs/users/create.md)
```

Comment (DebbieAtSeam) [inline docs/SUMMARY.md:171]:
```suggestion
    * [Create an ACS User](api/acs/users/create.md)
```

File: docs/SUMMARY.md [hunk 4]
```diff
@@ -162,40 +162,40 @@
...
+    * [List Users](api/acs/users/list.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:172]:
```suggestion
    * [List ACS Users](api/acs/users/list.md)
```

File: docs/SUMMARY.md [hunk 5]
```diff
@@ -162,40 +162,40 @@
...
+    * [Get a User](api/acs/users/get.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:173]:
```suggestion
    * [Get an ACS User](api/acs/users/get.md)
```

File: docs/SUMMARY.md [hunk 6]
```diff
@@ -162,40 +162,40 @@
...
+    * [Update a User](api/acs/users/update.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:174]:
```suggestion
    * [Update an ACS User](api/acs/users/update.md)
```

File: docs/SUMMARY.md [hunk 7]
```diff
@@ -162,40 +162,40 @@
...
+    * [Suspend a User](api/acs/users/suspend.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:175]:
```suggestion
    * [Suspend an ACS User](api/acs/users/suspend.md)
```

File: docs/SUMMARY.md [hunk 8]
```diff
@@ -162,40 +162,40 @@
...
+    * [Unsuspend a User](api/acs/users/unsuspend.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:176]:
```suggestion
    * [Unsuspend an ACS User](api/acs/users/unsuspend.md)
```

File: docs/SUMMARY.md [hunk 9]
```diff
@@ -162,40 +162,40 @@
...
+    * [Delete a User](api/acs/users/delete.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:177]:
```suggestion
    * [Delete an ACS User](api/acs/users/delete.md)
```

File: docs/SUMMARY.md [hunk 10]
```diff
@@ -162,40 +162,40 @@
...
+    * [Add a User to an Access Group](api/acs/users/add\_to\_access\_group.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:178]:
```suggestion
    * [Add an ACS User to an Access Group](api/acs/users/add\_to\_access\_group.md)
```

File: docs/SUMMARY.md [hunk 11]
```diff
@@ -162,40 +162,40 @@
...
+    * [Remove a User from an Access Group](api/acs/users/remove\_from\_access\_group.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:179]:
```suggestion
    * [Remove an ACS User from an Access Group](api/acs/users/remove\_from\_access\_group.md)
```

File: docs/SUMMARY.md [hunk 12]
```diff
@@ -162,40 +162,40 @@
...
+  * [Entrances](api/acs/entrances/README.md)
+    * [List Entrances](api/acs/entrances/list.md)
+    * [Get an Entrance](api/acs/entrances/get.md)
+    * [List Credentials with Access to an Entrance](api/acs/entrances/list\_credentials\_with\_access.md)
+  * [Access Groups](api/acs/access\_groups/README.md)
+    * [List Access Groups](api/acs/access\_groups/list.md)
+    * [Get an Access Group](api/acs/access\_groups/get.md)
+    * [Add a User to an Access Group](api/acs/access\_groups/add\_user.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:187]:
```suggestion
    * [Add an ACS User to an Access Group](api/acs/access\_groups/add\_user.md)
```

File: docs/SUMMARY.md [hunk 13]
```diff
@@ -162,40 +162,40 @@
...
+    * [Remove a User from an Access Group](api/acs/access\_groups/remove\_user.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:188]:
```suggestion
    * [Remove an ACS User from an Access Group](api/acs/access\_groups/remove\_user.md)
```

File: docs/SUMMARY.md [hunk 14]
```diff
@@ -162,40 +162,40 @@
...
+    * [List Users in an Access Group](api/acs/access\_groups/list\_users.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:189]:
```suggestion
    * [List ACS Users in an Access Group](api/acs/access\_groups/list\_users.md)
```

File: docs/SUMMARY.md [hunk 15]
```diff
@@ -162,40 +162,40 @@
...
+  * [Credentials](api/acs/credentials/README.md)
+    * [Create a Credential for a User](api/acs/credentials/create.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:191]:
```suggestion
    * [Create a Credential for an ACS User](api/acs/credentials/create.md)
```

File: docs/SUMMARY.md [hunk 16]
```diff
@@ -162,40 +162,40 @@
...
+    * [Assign a Credential to a User](api/acs/credentials/assign.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:192]:
```suggestion
    * [Assign a Credential to an ACS User](api/acs/credentials/assign.md)
```

File: docs/SUMMARY.md [hunk 17]
```diff
@@ -162,40 +162,40 @@
...
+    * [List Credentials](api/acs/credentials/list.md)
+    * [Get a Credential](api/acs/credentials/get.md)
+    * [Update a Credential](api/acs/credentials/update.md)
+    * [Unassign a Credential from a User](api/acs/credentials/unassign.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:196]:
```suggestion
    * [Unassign a Credential from an ACS User](api/acs/credentials/unassign.md)
```

**Review by DebbieAtSeam — APPROVED** [review]

File: docs/SUMMARY.md [hunk 18]
```diff
@@ -162,40 +162,40 @@
...
-  * [Credentials](api-clients/acs/credentials/README.md)
-    * [Create a Credential for a User](api-clients/acs/credentials/create.md)
-    * [Assign a Credential to a User](api-clients/acs/credentials/assign.md)
-    * [List Credentials](api-clients/acs/credentials/list.md)
-    * [Get a Credential](api-clients/acs/credentials/get.md)
-    * [Update a Credential](api-clients/acs/credentials/update.md)
-    * [Unassign a Credential from a User](api-clients/acs/credentials/unassign.md)
-    * [Delete a Credential](api-clients/acs/credentials/delete.md)
-    * [List Accessible Entrances](api-clients/acs/credentials/list\_accessible\_entrances.md)
+  * [Users](api/acs/users/README.md)
```
Comment (DebbieAtSeam) [inline docs/SUMMARY.md:170]:
```suggestion
  * [ACS Users](api/acs/users/README.md)
```

**Review by DebbieAtSeam — APPROVED** [review]

File: docs/api/acs/access_groups/add_user.md (hunk 1 above)
Comment (razor-x) [reply docs/api/acs/access_groups/add_user.md:132]: That looks like a mistake!

File: docs/api/acs/systems/README.md (hunk 2 above)
Comment (razor-x) [reply docs/api/acs/systems/README.md:106]: I don't know if we have a choice right now. I don't want to block on it.

I want to support annotating the enums though somehow in the zod descriptions like

```
---
legic_connect:
  undocumented: ...
```

Comment (razor-x) [reply docs/api/acs/systems/README.md:106]: https://github.com/seamapi/blueprint/issues/128

File: docs/SUMMARY.md (hunk 3 above)
Comment (razor-x) [inline docs/SUMMARY.md]:
```suggestion
    * [Create an ACS User](api/acs/users/create.md)
```

**Comment by razor-x** [comment]
@DebbieAtSeam Feel free to merge this first thing tomorrow!

**Pull request merged**